frontend/express/public/stylesheets/styles/manifest.scss

```diff
@@ -29,4 +29,7 @@
 @use "states/state";
 
 // All core "Plugins" views specific SASS partials
-@use "../../core/user-activity/stylesheets/main"
+@use "../../core/user-activity/stylesheets/main" as user-activity-main-style;
+@use "../../core/session-overview/stylesheets/main" as session-overview-main-style;
+@use "../../core/session-durations/stylesheets/main" as session-durations-main-style;
+@use "../../core/session-frequency/stylesheets/main" as session-frequency-main-style;
```

Comment:
for all the imported scss partials, i dont think we need a namespace for them.

Reply:
the compiler complains because of the same file name being used, hence I had to namespace it.